control/mpc_follower/src/qp_solver/qp_solver_osqp.cpp

```diff
@@ -14,9 +14,8 @@
  * limitations under the License.
  */
 #include "mpc_follower/qp_solver/qp_solver_osqp.h"
-#include <ros/ros.h>
 
-QPSolverOSQP::QPSolverOSQP() {}
+QPSolverOSQP::QPSolverOSQP(const rclcpp::Logger & logger) : logger_{logger} {}
```

Comment:
The  `rclcpp::get_logger("name")` seems to be available, so let's use it to remove the logger arg. 
I just tested in the pacmod_interface and it looks working correctly.

In the cpp file
```
RCLCPP_INFO(this->get_logger(), "test using this->get_logger()");
RCLCPP_INFO(rclcpp::get_logger("NAME"), "test using rclcpp::get_logger(std::string name)");
```

Then output is 
```
[pacmod_interface-1] [INFO] [1603958623.148726695] [pacmod_interface]: test using this->get_logger()
[pacmod_interface-1] [INFO] [1603958623.148794838] [NAME]: test using rclcpp::get_logger(std::string name)
```

Maybe we can use this here like `rclcpp::get_logger("QPSolverOSQP")`